CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - executions and granules list calls updated to search un-archived records
 - **CUMULUS-4223**
   - Upgrade cypress docker image, folders, test specs, configs to v12.17.4
+- **CUMULUS-3627**
+  - Added `INITIAL_TIMEZONE_FORMAT` environment variable to allow control over table date/time column's timezone format 
```

Comment:
`over table date/time column's timezone format ` -> `over the table's date/time timezone format`